### PR TITLE
Removed debug flag from template

### PR DESCRIPTION
**Pull request by timwright12**

### CODE REVIEW

**Review by kelsonic — APPROVED** [review]
Thanks! 🙂

File: src/site/includes/debug.drupal.liquid [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% if buildtype != 'vagovprod' and omitdebug == false %}
+{% if buildtype != 'vagovprod' %}
```
Comment (ncksllvn) [inline src/site/includes/debug.drupal.liquid:1]:
```suggestion
{% if isPreview or buildtype != 'vagovprod' and omitdebug == false %}
```

It doesn't hurt to include this in the prod preview server IMO!  Double check my boolean arithmetic in that if-statement though (not certain how Liquid will interpret this)

Comment (kelsonic) [reply src/site/includes/debug.drupal.liquid:1]: True! 🙂 I think you will need parentheses, or at least my brain needs them 😂 

```liquid
{% if isPreview or (buildtype != 'vagovprod' and omitdebug == false) %}
```

Comment (ncksllvn) [reply src/site/includes/debug.drupal.liquid:1]: I was wondering if Liquid supports the parentheses lol

**Pull request merged**